tests/projects/base_pipeline.py

```diff
@@ -29,6 +30,12 @@ def setup_method(self, method):
             / "assets"
         )
 
+    def teardown_method(self, method):
+        # revert change of default project after project creation
+        if self.project:
+            mlrun.mlconf.default_project = "default"
+            pipeline_context.set(None)
```

Comment:
not sure i understand why it is required

Reply:
pipeline_context is a global variable that is being set when creating or setting a project, it is being accessed when  running functions / workflows without running it from the project object but rather straight from the relevant object.

Though I am not sure why not applying that cleanup as part of the `config_test_base` in `common_fixture.py` that way it will automatically apply on all tests